src/app/eyrie/eyrie.component.html

```diff
@@ -192,7 +192,8 @@ <h2>{{ 'Sections.Evening' | translate }}</h2>
       <ion-card-content>
         <ol>
           <li class="rule-message" *ngFor="let message of bot.evening(translateService)">
-            <span [innerHTML]="rendererService.formatString(message)"></span>
+            <span *ngIf="message.meta.type != 'score'" [innerHTML]="rendererService.formatString(message.text)"></span>
+            <a *ngIf="message.meta.type == 'score'" [innerHTML]="rendererService.formatString(message.text)" (click)="bot.addVP(message.meta.value)"></a>
```

Comment:
ideally, this would be a reusable component that each class can use as needed (otherwise it's going to be hard to change it globally). it should _probably_ also be a button instead of a link so users know that it does something. if you do go the route of making it a button, feel free to cannibalize the styles and make it thin and inline with the text so it doesn't push the text too far up or down.

if it were componentized, it would also offer the ability to block the action if it was clicked. since it should only happen once per action, it would be good to block so users don't click it multiple times.

like you suggest, it should pop up a Toast to let the user know the action happened. in this toast, an undo button could be added in case the user mistakenly clicked it.